File: dataset_reduzido/codenet/Java/Code/s383412435.java
import java.util.Scanner;
import java.util.Arrays;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        long a[] = new long[N];
        int tmp;
        int ans = 0;
        for (int z = 0; N > z; z++) {
            a[z] = sc.nextInt();
        }
        Arrays.sort(a);
        if (N == 1 && a[0] > 1) {
            ans = 1;
        } else {
            for (int j = 0; N > j; j++) {
                if (j == 0 || a[j] != a[j - 1]) {
                    tmp = 1;
                    for (int i = j+1; N > i; i++) {
                        if (a[j] == a[i]) {
                            tmp++;
                        }
                        else{
                            break;
                        }
                    }
                    
                    if (tmp > a[j]) {
                        ans += tmp - a[j];
                    } else if (tmp < a[j]) {
                        ans += tmp;
                    }
                
                }
            }
        }
        System.out.println(ans);
    }
}

File: output/Llama-2-100/codenet/Java/Python/s383412435.py
def main():
    N = int(input())
    a = [int(input()) for _ in range(N)]
    ans = 0
    if N == 1 and a[0] > 1:
        ans = 1
    else:
        for j in range(N):
            if j == 0 or a[j] != a[j-1]:
                tmp = 1
                for i in range(j+1, N):
                    if a[j] == a[i]:
                        tmp += 1
                    else:
                        break
                if tmp > a[j]:
                    ans += tmp - a[j]
                else:
                    ans += tmp
    print(ans)
